File: ios/GaleriaOverlayModule.swift
import ExpoModulesCore

public class GaleriaOverlayModule: Module {
    public func definition() -> ModuleDefinition {
        Name("GaleriaOverlay")

        View(GaleriaOverlayView.self) {
            Prop("visible") { (view, visible: Bool?) in
                view.visible = visible ?? false
            }

            Prop("showAfterOpen") { (view, show: Bool?) in
                view.showAfterOpen = show ?? false
            }
        }
    }
}

File: android/src/main/java/nandorojo/modules/galeria/GaleriaOverlayModule.kt
package nandorojo.modules.galeria

import expo.modules.kotlin.modules.Module
import expo.modules.kotlin.modules.ModuleDefinition

class GaleriaOverlayModule : Module() {
    override fun definition() = ModuleDefinition {
        Name("GaleriaOverlay")

        View(GaleriaOverlayView::class) {
            Prop("visible") { view: GaleriaOverlayView, visible: Boolean ->
                view.visible = visible
            }
        }
    }
}
